data/aviation_data.csv

```diff
@@ -0,0 +1,274 @@
+country_name,icao,icaoregion,statfor,pru
```

Comment:
What is the rationale for not including those directly to the original countrycode_data.csv?